cmd/goose/main.go

```diff
@@ -7,12 +7,18 @@ import (
 	"io/fs"
 	"log"
 	"os"
+	"path/filepath"
 	"runtime/debug"
+	"sort"
 	"strconv"
+	"strings"
+	"text/tabwriter"
 	"text/template"
 
 	"github.com/pressly/goose/v3"
 	"github.com/pressly/goose/v3/internal/cfg"
+	"github.com/pressly/goose/v3/internal/migrationstats"
```

Comment:
It's possible folks might want this functionality exposed in the goose package, if so, we can move this to the top-level or create a `pkg` with helpful utilities that are not related to migration logic per se.